packages/legend-query-builder/src/graphManager/QueryBuilderSupportedFunctions.ts

```diff
@@ -65,3 +65,95 @@ export enum QUERY_BUILDER_SUPPORTED_FUNCTIONS {
   SUM = 'meta::pure::functions::math::sum',
   UNIQUE_VALUE_ONLY = 'meta::pure::functions::collection::uniqueValueOnly',
 }
+
+// TDS
+export const DEFAULT_POST_FILTER_LAMBDA_VARIABLE_NAME = 'row';
+export const TDS_ROW = 'meta::pure::tds::TDSRow';
+
+export const QUERY_BUILDER_PROPERTY_SEARCH_MAX_DEPTH = 5;
+export const QUERY_BUILDER_PROPERTY_SEARCH_RESULTS_LIMIT = 100;
+export enum QUERY_BUILDER_PROPERTY_SEARCH_TYPE {
+  CLASS = 'CLASS',
+  STRING = 'STRING',
+  BOOLEAN = 'BOOLEAN',
+  NUMBER = 'NUMBER',
+  DATE = 'DATE',
+}
+
+export enum QUERY_BUILDER_HASH_STRUCTURE {
```

Comment:
Let's move all hash stuff to `graphManager/QueryBuilderHashUtils.ts`. Also, all of the `DEFAULT_POST_FILTER_LAMBDA_VARIABLE_NAME`, .. that you added here, I think they belongs to `QueryBuilderConfig` already